storage/schema.org.md

```diff
@@ -0,0 +1,230 @@
+# REST API using Schema.org
+
+Request the root:
+
+```
+GET /files HTTP/2
+accept: application/ld+json
+```
+
+```
+HTTP/2 200
+content-type: application/ld+json
+
+{
+    "@context": "http://schema.org",
+    "@type": "WebAPI",
+    "potentialAction": [
+        {
+            "@type": "CreateAction",
+            "target": {
+                "@type": "EntryPoint",
+                "httpMethod": "POST",
+                "urlTemplate": "/files/actions",
+                "encodingType": "multipart/related; type=application/ld+json",
```

Comment:
`encodingType` and `contentType` are input vs output formats?

Reply:
Yep. More in https://schema.org/docs/actions.html; not sure output details are that useful.

Reply:
(Though this is for `Accept`.)